pdr_backend/lake/etl.py

```diff
@@ -193,12 +251,9 @@ def get_timestamp_values(
         self, table_names: List[str], default_timestr: str
     ) -> UnixTimeMs:
         max_timestamp_values = self._get_max_timestamp_values_from(
-            [NamedTable(tb) for tb in table_names]
+            [Table(tb) for tb in table_names]
         )
 
-        logger.info(
```

Comment:
why was this logging line removed?

Reply:
because I am a heavy user of the lake CLI and the lake CLI is incomprehensible

it generates a ton of repeating/redundant/granular logs that bloats the output

so i removed a bunch of it and made sure that you can still follow all core logic.